packages/lab/src/index.ts

```diff
@@ -103,7 +103,7 @@ const plugin: JupyterFrontEndPlugin<IRisePreviewTracker> = {
     }
 
     const factory = new RisePreviewFactory(getRiseUrl, commands, {
-      name: 'rise',
+      name: 'Rise Slides',
```

Comment:
FYI: There is a new `label` field for the displayed name in Lab 4

```suggestion
      name: 'rise',
      label: trans.__('Rise Slides'),
```

Xref: 
- https://jupyterlab.readthedocs.io/en/latest/api/interfaces/docregistry.DocumentRegistry.IWidgetFactoryOptions.html#label
- https://github.com/mwouts/jupytext/blob/28cc7de53d403838caf24c3470df95e94a82d132/packages/labextension/src/index.ts#L530